src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/AttackSystem.java

```diff
@@ -55,24 +53,23 @@
     @In
     EntityManager entityManager;
     @In
-    private LocalPlayer localPlayer;
-    @In
     private WorldProvider worldProvider;
     @In
     private BlockManager blockManager;
 
     private EntityRef flagSlot;
     private EntityRef item;
 
-
     @ReceiveEvent(components = {FlagDropOnActivateComponent.class, PlayerCharacterComponent.class, HasFlagComponent.class})
     public void onActivate(ActivateEvent event, EntityRef entity) {
         dropFlagOnPlayerAttack(event, entity);
     }
 
-    /** When player activates another with the magic staff, checks to see if the attacked player has a flag
+    /**
+     * When player activates another with the magic staff, checks to see if the attacked player has a flag
      * If so, makes the player drop the flag
-     * targetPlayer is player being attacked */
+     * targetPlayer is player being attacked
```

Comment:
Change this to
`@param targetPlayer The player being attacked`
Even if this is just a private method and you are not annotating every single parameter, might as well annotate the ones we are doing properly :)